src/Data/Number.purs

```diff
@@ -5,12 +5,34 @@ module Data.Number
   , isNaN
   , infinity
   , isFinite
+  , readInt
+  , readFloat
```

Comment:
I don't think it makes sense to export this function, as it's just a less safe version of `fromString`.

Reply:
Agreed.